scipy/ndimage/tests/test_morphology.py

```diff
@@ -2265,6 +2265,7 @@ def test_grey_erosion01(self, xp):
                                               [5, 5, 3, 3, 1]]))
 
     @skip_xp_backends("jax.numpy", reason="output array is read-only.")
+    @skip_xp_backends("dask.array", reason="output array is read-only.")
     @xfail_xp_backends("cupy", reason="https://github.com/cupy/cupy/issues/8398")
     def test_grey_erosion01_overlap(self, xp):
```

Comment:
Would be interested to know why having an `output` array is a requirement here

Is this test specifically trying to test in-place operations? If so, maybe it is worth clarifying a comment or the test name